openmetadata-ui/src/main/resources/ui/src/components/AppRouter/AuthenticatedAppRouter.tsx

```diff
@@ -286,7 +286,7 @@ const AuthenticatedAppRouter: FunctionComponent = () => {
       <Route
         exact
         component={BulkEntityImportPage}
-        path={ROUTES.ENTITY_IMPORT}
+        path={[ROUTES.ENTITY_IMPORT, ROUTES.BULK_EDIT_ENTITY_WITH_FQN]}
```

Comment:
ques - will bulk edit be open for editing by all users, do we need a permission check ??